File: backend/internal/model/wms.go
package model

type Wms struct {
	ID          int      `json:"id"`
	Name        string   `json:"name"`
	Description string   `json:"description"`
	Layers      []string `json:"layers"`
	Url         string   `json:"url"`
	Version     string   `json:"version"`
	IsActive    bool     `json:"is_active"`
	AuthType    string   `json:"auth_type"`
	Username    string   `json:"username"`
	Password    string   `json:"password"`
}

File: backend/src/domain/wms.rs
pub struct Wms {
    id: i32,
    name: String,
    description: String,
    layers: Vec<String>,
    url: String,
    version: String,
    is_active: bool,
    auth_type: String,
    auth_username: String,
    auth_password: String,
}
